src/main/resources/org/sagebionetworks/web/client/widget/entity/menu/v2/ActionMenuWidgetViewImpl.ui.xml

```diff
@@ -17,24 +17,22 @@
 		<b:Tooltip title="Show query">
 			<a:ActionButton visible="false" icon="DATABASE" addStyleNames="margin-right-5" action="SHOW_QUERY" />
 		</b:Tooltip>
-		<b:Tooltip title="Edit query results">
-			<a:ActionButton visible="false" icon="EDIT" addStyleNames="margin-right-5" action="EDIT_TABLE_DATA" />
+		<b:Tooltip ui:field="editTableDataTooltip" title="Edit query results">
+            <bh:Span> <!-- Have to wrap the button in a span we get a mouseevent to show the tooltip if the button is disabled -->
```

Comment:
> Have to wrap the button in a span we get a mouseevent to show the tooltip if the button is disabled

Interesting!
Seems to be a minor spacing issue here which you could optionally fix.